### ISSUE TITLE
What if a solution consists of multiple binaries of different programming langauges?

### ISSUE THREAD

**Issue by nya3jp** [icfpcontest2020/dockerfiles#3]
If I understand README correctly, this repository is going to host Dockerfiles, each of which corresponds to a single programming language.

But what if a solution consists of multiple binaries of different programming languages? For example, what if it a solution consists of C++ binary and Rust binary interacting each other with IPC?

**Comment by beevee** [comment]
As a part of upcoming contest you will be required to submit a small program using a single programming language. I cannot disclose the nature of this task in advance, but we think that given the constraints you would not want to write this small program as a set of multiple binaries using multiple runtimes.

Of course, you can still use any languages locally during development of this small program.

**Comment by igorlukanin** [comment]
@beevee & @nya3jp, as we now have support for [bash](https://github.com/icfpcontest2020/dockerfiles/tree/master/dockerfiles/bash), is this issue still relevant?

**Comment by nya3jp** [comment]
My initial question was resolved by the answer by @beevee. I don't know the nature of the task (of course!), so I just trust the reply that participants are unlikely to write multi-language solutions.

You can close the issue now, but keeping it open for a while might be useful for other participants who got the same question.

**Comment by beevee** [comment]
Exactly, I'm keeping it open to help other users who might have the same question.

**Comment by zloyrusskiy** [comment]
What about dependencies which builded with another language/libraries requirements? (e.g. opencv)

Maybe it’s worth letting participants specify their own Dockerfile with a build time limit?
